#! /bin/sh /usr/share/dpatch/dpatch-run
## 04_default_terminal.dpatch by Andrew Lee <andrew@linux.org.tw>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Support x-terminal-emulator as default terminal

@DPATCH@
diff -urNad pcmanfm-0.5~/src/pref-dialog.c pcmanfm-0.5/src/pref-dialog.c
--- pcmanfm-0.5~/src/pref-dialog.c	2008-07-15 07:58:33.000000000 +0800
+++ pcmanfm-0.5/src/pref-dialog.c	2008-09-26 10:18:01.000000000 +0800
@@ -66,6 +66,7 @@
 static const int small_icon_sizes[] = { 48, 36, 32, 24, 20, 16, 12 };
 static const char* terminal_programs[] =
 {
+    "x-terminal-emulator",
     "lxterminal",
     "aterm",
     "urxvt",
